packages/grafana-data/src/types/panel.ts

```diff
@@ -28,6 +28,14 @@ export interface PanelData {
   /** Contains data frames with field overrides applied */
   series: DataFrame[];
 
+  /**
+   * This flag will change when the structure is different from the previous response
+   * The structure includes frames, names, types, and any config properties
+   *
+   * 0 or undefined indicates that the change status is unknown or not computed
```

Comment:
Is this true? Looking at the code it seems like the value can be either `undefined` or >1000

Reply:
FYI: removing the changes to property for now.  Lets start with a utility function, and perhaps add it as a higher level property if it is more widely useful and efficient